staging/src/k8s.io/apiserver/pkg/cel/openapi/maplist_test.go

```diff
@@ -0,0 +1,297 @@
+/*
```

Comment:
Curious what others think, but I think I would prefer keeping this new maplist_test.go file but deleting staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/maplist_test.go. While it is good to sanity check that the translation layer is working, I feel like having essentially the same test suite in duplicate will cause drift.

WDYT @liggitt ?

The translation layer changes look good to me otherwise.

Reply:
Let's keep the tests right now to probe this refactor does not break existing features. We can always remove the tests in another PR.